src/hapi/pipelines/configs/operational_presence.yaml

```diff
@@ -1,83 +1,81 @@
 # TODO: combine all OP scrapers once metadata issue solved
-operational_presence_afg:
-  dataset: "afghanistan-who-does-what-where-january-to-march-2023"
-  resource: "afghanistan-3w-operational-presence-january-march-2023.csv"
-  format: "csv"
-  headers: 2
-  source_date:
-    start: "01/01/2023"
-    end: "31/03/2022"
-  input:
-    - "#adm2+code"
-    - "#org +name"
-    - "#org +acronym"
-    - "#org +type +name"
-    - "#sector +cluster +name"
-  output:
-    - "adm2_code"
-    - "org_name"
-    - "org_acronym"
-    - "org_type_name"
-    - "sector_name"
-  output_hxl:
-    - "#adm2+code"
-    - "#org+name"
-    - "#org+acronym"
-    - "#org+type+name"
-    - "#sector+name"
+operational_presence_admintwo:
```

Comment:
Can this be a configurable scraper @mcarans?  The reason I made it custom was because the configurable version produced results with only one row per admin unit, but we need many rows per admin unit.

Reply:
Thanks @b-j-mills . I will check that.

Reply:
I've created https://humanitarian.atlassian.net/browse/HAPI-209

Reply:
I have added that functionality to HDX Python Scraper and closed the ticket.